sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_v1/query.sql

```diff
@@ -1,3 +1,6 @@
+-- Should this be a persisted UDF? tier1_country
+-- definition probably does not change between
+-- specific queries.
```

Comment:
let's leave this as is, tier1 country can have different definitions based on context (vpn's tier1 is not necessarily firefox desktop's tier1) and formatting (e.g. country name vs country code)